src/main.py

```diff
@@ -84,6 +84,14 @@ def get_cmdline_parser() -> argparse.ArgumentParser:
         default="c-cpp",
         help="Language of project"
     )
+    report_parser.add_argument(
+        "--analyses-with-json",
```

Comment:
Could we change this to something like `output-json`?

Reply:
Done